src/style/sass/main.scss

```diff
@@ -64,7 +78,7 @@ body {
                     margin-top: 20px;
                 }
                 
-                .uiButton {
+                .uiButton, #filterButton, #filterInput {
```

Comment:
why add ids to the scss?

Reply:
They're unique elements. The only components with a "filtering" related function, but I wanted their padding and margin to match.